openapi/v2/support-scheduler.yaml

```diff
@@ -177,12 +246,8 @@ components:
         target:
           description: "The target of the action"
           type: string
-        topic:
-          description: "When the action supports pub/sub the topic to which a message should be published."
-          type: string
-        user:
-          description: "If authentication is required, the username"
-          type: string
+      required:
+        - name
```

Comment:
Same question on name or id is required.

Reply:
Modified. Name or id is required

Reply:
Add to description to clarify that this is an either requirement, not both

Reply:
Added description `either id or name should be specified`.